### ISSUE TITLE
Closing with <esc> stopped working

### ISSUE THREAD

**Issue by kuba-gaj** [abenz1267/walker#69]
as in the title, in both server and standalone modes

**Comment by abenz1267** [comment]
Can't reproduce. What version are you on? You need to give a bit more info.

**Comment by kuba-gaj** [comment]
version 0.3.5
I'm sure it used to work. I updated and got default configs from repo to get new stuff

https://github.com/user-attachments/assets/535fb92f-9a84-4dba-b107-f5bfe8ab3223

**Comment by omnigenous** [comment]
I'm on same version, can't reproduce.

**Comment by kuba-gaj** [comment]
Ok I figured it out, looking into the code helped.

My main programmable keyboard is in the service, and I'm currently using my spare keyboard with [keyd](https://github.com/rvaiya/keyd) to replicate layers and mapping of `capslock` to `esc` on tap, and `lctrl` on press with another key. Looks like `walker` was putting me into `activation_mode` and out on key press. I turned on `use_alt` in config and the issue is gone.

Btw, looks like walker fails to start and create default configs if `~/.config/walker` dir does not exist.
